editor/src/components/navigator/navigator-conditionals.spec.browser2.tsx

```diff
@@ -113,6 +118,9 @@ function dragElement(
       }),
     )
 
+    await wait(1) // FIXME: this might be necessary for react-dnd internal delays
+    await midDragCallback()
```

Comment:
isn't the problem here that the code above is not using using an async/await act for wrapping fireEvent? see for example https://github.com/concrete-utopia/utopia/blob/03dec098e333408a5f80ac410b38fe64d2345458/editor/src/components/canvas/event-helpers.test-utils.tsx#L67-L80

Reply:
thanks, that worked, added it in https://github.com/concrete-utopia/utopia/pull/3581/commits/7b9ee78eb61f30e683189edfd6011ac7ebc5abe0! (Good to know this little gotcha 🙃)

However I still needed to add a `wait` in the test for sideways drag (to reparent up the hierarchy)